.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
```

Comment:
```suggestion
```

GitLens is really depends on personal preference and is not required by the project. I don't use it for example

Reply:
Sure, fixed